cheroot/connections.py

```diff
@@ -99,76 +99,87 @@ def expire(self):
         # find any connections still registered with the selector
         # that have not been active recently enough.
         threshold = time.time() - self.server.timeout
-        timed_out_connections = (
+        timed_out_connections = [
             (sock_fd, conn)
             for _, (_, sock_fd, _, conn) in self._selector.get_map().items()
             if conn != self.server and conn.last_used < threshold
-        )
+        ]
```

Comment:
The outer generator expression may be kept given that we'll make the iterable target immutable by using `.copy()` early.

Reply:
Also, I think all of the `get_map()` calls should be followed by `.copy()` as long as it's used in a threaded env: https://github.com/cherrypy/cheroot/issues/312#issuecomment-673477303.